camayoc/tests/qpc/conftest.py

```diff
@@ -2,13 +2,10 @@
 import pytest
 
 from camayoc import api
-from camayoc.qpc_models import (
-    Credential,
-    Scan,
-    Source,
-)
+from camayoc.qpc_models import Credential, Scan, Source
 
 
+@pytest.fixture(scope='function')
 def sort_and_delete(trash):
```

Comment:
this is connected to the change with the cleanup fixture. I think this is going to break other tests